anycloud/node/README.md

```diff
@@ -1,48 +1,6 @@
 # AnyCloud
 
-AnyCloud is a Lambda alternative for Node.js that works with multiple cloud providers. We’re building the features of Lambda while fixing the problems we have run into while using it. Namely:
```

Comment:
This is the file that is rendered by NPM itself. Are you sure you want to gut it?

Reply:
Yes. I prefer to centralize the documentation and information so we don't have to be constantly changing this. People won't really land on the npmjs site without reading about it somewhere else. Also things will remain just one click away.